src/components/MapView/SimpleDropdown/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { FormControl, MenuItem, Select } from '@material-ui/core';
+import React from 'react';
+
+type OptionValue = number | string;
+type OptionLabel = string;
+type DropdownOption = [OptionValue, OptionLabel];
+
+interface SimpleDropdownProps {
+  options: DropdownOption[];
+  value: OptionValue;
+  // want to do (val: OptionValue)
+  // but this causes type conflicts when a user knows that val will be an enum
+  onChange: (val: any) => void;
+}
+
+export default function SimpleDropdown({
+  options,
+  value = '',
+  onChange,
+  ...rest
+}: SimpleDropdownProps) {
+  return (
+    <FormControl {...rest}>
+      <Select
+        value={value}
+        onChange={(e: any) => {
```

Comment:
I refactored OptionValue to be a constrained generic instead of a static type and so we can just remove any here and use the React.ChangeEvent type assigned by default.